File: URI/1 - INICIANTE/C/2028 - SequenciaDeSequencia.c
#include <stdio.h>

int main(){
	int num, cont = 0, y, z, caso = 0;
	
	while(scanf(" %d", &num) != EOF){
		cont = 1;
		caso ++;
		for(y = 0; y < num+1; y++){
			for(z = 0; z < y; z++){
				cont++;
			}
		}
		if(cont == 1){
			printf("Caso %d: %d numero\n",caso , cont);
		}else{
			printf("Caso %d: %d numeros\n",caso , cont);
		}
		if(num == 0){
			printf("0\n");
		}else{
			printf("0 ");
		}
		for(y = 0; y < num+1; y++){
			for(z = 0; z < y; z++){
				if(y == num && z == num - 1){
					printf("%d\n", y);
				}else{
					printf("%d ", y);
				}
				
			}
		}
		printf("\n");
	}
	
	return 0;
}

File: URI/1 - INICIANTE/Python/2028 - SequenciaDeSequencia.py
caso = 0
while True:
    try:
        num = int(input())
        cont = 1
        caso += 1
        for y in range(0, num+1):
            for z in range(0, y):
                cont += 1
        if cont == 1:
            print('Caso {}: {} numero'.format(caso, cont))
        else:
            print('Caso {}: {} numeros'.format(caso, cont))
        if num == 0:
            print('0')
        else:
            print(0,end=' ')
        for y in range(0, num+1):
            for z in range(0, y):
                if y == num and z == num - 1:
                    print(y)
                else:
                    print(y, end=' ')
        print('')
    except:
        break